src/main/java/core/basesyntax/UserService.java

```diff
@@ -2,10 +2,18 @@
 
 public class UserService {
     public void registerUser(User user) {
-
+        PasswordValidator passwordValidator = new PasswordValidator();
+        try {
+            passwordValidator.validate(user.getPassword(), user.getRepeatPassword());
+        } catch (PasswordValidationException pas) {
+            System.out.println("Your passwords are incorrect. Try again.");
+            return;
+        }
+        UserService userService = new UserService();
```

Comment:
do we need to create the object of UserService class?

Reply:
done